### PR TITLE
Explanation of convenience PIN

### PR DESCRIPTION
**Pull request by rui0122**
None of public documents defines what convenience PIN is.

## Why

- Closes #[Issue Number]

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-faq.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ sections:
...
+          A Convenience Pin is used for sign-in (Windows Hello gesture only) which includes PIN, and Biometrics sign-in to the PC is not compatible with WHFB or with Hybrid AAD Join. Microsoft is committed to its vision of a <u>world without passwords.</u> We recognize the *convenience* provided by convenience PIN, but it stills uses a password for authentication. Microsoft recommends that customers using Windows 10 and convenience PINs should move to Windows Hello for Business. New Windows 10 deployments should deploy Windows Hello for Business and not convenience PINs. Microsoft will be deprecating convenience PINs in the future and will publish the date early to ensure customers have adequate lead time to deploy Windows Hello for Business.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-faq.yml:41]:
```suggestion
          A convenience PIN is used for sign-in (Windows Hello gesture only) which includes a PIN, and biometrics sign-in to the device is not compatible with Windows Hellow for Business or with Hybrid Azure AD join. Microsoft is committed to its vision of a <u>world without passwords.</u> We recognize the *convenience* provided by convenience PIN, but it still uses a password for authentication. We recommend that customers using Windows 10 and convenience PINs move to Windows Hello for Business. New Windows 10 deployments should deploy Windows Hello for Business, not convenience PINs. Microsoft will be deprecating convenience PINs in the future and will publish the date early to ensure customers have adequate lead time to deploy Windows Hello for Business. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rui0122, kindly commit the suggested changes. Many thanks.

**Review by rui0122 — COMMENTED** [review]
Submit

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by paolomatarazzo** [comment]
@rui0122 thank you for the contribution. However, the suggested sentence doesn't provide an explanation of what *convenience PIN* is. To me, it actually adds more confusion. Can you please reword the sentence so that I can understand better what you're trying to communicate? "A convenience PIN is used for sign-in (Windows Hello gesture only) which includes a PIN, and biometrics sign-in to the device is not compatible with Windows Hellow for Business or with Hybrid Azure AD join."

**Comment by paolomatarazzo** [comment]
I updated the description. Note that this Q/A refers about deprecation of convenience PIN, not about its description. (similar to the Q/A above). I'll close this PR and update the answer on a separate, existing PR.

**Pull request closed without merging**